cypress/e2e/item/upload/dropzoneUpload.cy.ts

```diff
@@ -0,0 +1,29 @@
+import { buildItemPath } from '@/config/paths';
+import { DROPZONE_HELPER_ID } from '@/config/selectors';
+
+import { SAMPLE_ITEMS, SAMPLE_PUBLIC_ITEMS } from '../../../fixtures/items';
+
+describe('home screen - dropzone helper visibility', () => {
```

Comment:
for this I would wrap both tests in a describe that is called 'Dropzone Helper' and then have the two describe say 'Home Screen' & 'Empty Folder'.